index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="desktop.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div > 
+ <header class="header" id="mylogo-section">
+    <div class="name">
+        <a  class="mylogo" href="#">My logo</a>
+    </div>
+        <nav >
+            <a href="#"><img  class="icon-menu" id="logoburger" src="./image/logobuger.JPG" alt=""></a>
+            <ul class="menu">
+                <a class="Portfolio-desc" href="#Portfolio">Portfolio</a>
+                <a class="About-desc" href="#aboutcardid">About</a>
+                <a class="contact-desc" href="#contactcard">Contact</a>
+             </ul>
+        </nav>
+</header>
+<section class="firstcardContain">
+            <div id="firstcardBackid" class="firstcardBack">
+                        <div class="mainiam">
+                            <h1 class="introduction" id="introduction"> I'm Mihertab Glad to see you!</h1>
+                        </div>
+                        <div class="section_01" id="section_description">
+                            <p class="about"> I’m a software developer! I can help you build a product , 
+                                feature or website Look through some of my work and experience! If you like what 
+                                you see and have a project you need coded, don’t hestiate to contact me. </p>
+                        </div>
+                        <div class="letconnect">
+                            <h4 id="connectmywork01" class="connectmywork_01"> LET’S CONNECT</h4>
+                        </div>
+                        <div class = "social-section_01">
+                            <ul class="social-icons-mywork_01">
+                                <li><a href="#"><img src="./image/Vectortweet.png" alt="Twiter"></a></li>
+                                <li><a href="#"><img src="./image/Linkedin.svg" alt="Linkdin"></a></li>
+                                <li><a href="#"><img src="./image/VectorM.jpg" alt="Medium"></a></li>
+                                <li><a href="#"><img src="./image/VectorGit.png" alt="Githab"></a></li>
+                                <li><a href="#"><img src="./image/Vector5.png" alt="angellist"></a></li>
+                            </ul>
+                        </div>
+                </div> 
+</section>
+<div class="menu-back" id="menubackpop">
+    <div id="xvalue" class="close-value">&#10005;
+       
+        <!-- <a ><button id="xvalue" class="xvalue">&#10005;<img  src="image/x-Icon.png" alt=""></button></a> -->
```

Comment:
Great job @DamenaM 🤝

- [ ] [Optional] Kindly remove commented code from your project as it is regarded as a bad practice.